python_apps/pypo/pypo/__main__.py

```diff
@@ -119,7 +120,7 @@ def configure_environment():
     consoleHandler.setFormatter(logFormatter)
     rootLogger.addHandler(consoleHandler)
 except Exception as e:
-    print("Couldn't configure logging", e)
+    print(("Couldn't configure logging", e))
```

Comment:
Parenthesis are not needed:

```suggestion
    print("Couldn't configure logging:", e)
```